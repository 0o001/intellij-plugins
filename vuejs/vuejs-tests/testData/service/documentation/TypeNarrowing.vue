<script setup lang="ts">
let <error>dummyErrorToSatisfyTestFramework</error>: string = 1;

/**
 * docs have to be removed!
 */
let local = 323 as number | string;
</script>

<template>
  <div v-if="typeof local === 'number'">
    <div v-text="<caret>local.toFixed()"/>
  </div>
</template>