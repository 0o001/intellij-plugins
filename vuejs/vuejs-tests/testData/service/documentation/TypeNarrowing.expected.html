(property) local: number<br/>[ide]<br/>let local: number